JPS_VIRTUALSENSOR/AermodAgent/docker/.m2/settings.xml

```diff
@@ -36,6 +41,17 @@
                         <enabled>true</enabled>
                     </snapshots>                    
                 </repository>
+                <repository>
+                    <id>stack-repo</id>
+                    <url>https://maven.pkg.github.com/TheWorldAvatar/stack</url>
```

Comment:
shouldn't this be https://maven.pkg.github.com/TheWorldAvatar? or https://maven.pkg.github.com/TheWorldAvatar/*? Can it download the base lib and parent pom?

Reply:
Parent pom still download from https://maven.pkg.github.com/cambridge-cares/TheWorldAvatar, have added https://maven.pkg.github.com/TheWorldAvatar/baselib for latest base lib.

Reply:
Updated to https://maven.pkg.github.com/TheWorldAvatar/*